cinema_service/settings.py

```diff
@@ -133,6 +133,10 @@
 
 STATIC_URL = "static/"
 
+MEDIA_ROOT = BASE_DIR / "media"
+
+MEDIA_URL = "/media/"
```

Comment:
Ensure that the `MEDIA_ROOT` and `MEDIA_URL` settings are correctly configured to handle image uploads via the `/upload-image/` endpoint. This is crucial for the functionality of the `ImageField` in the `Movie` model.